api/openapi-spec/swagger.json

```diff
@@ -6550,6 +6550,88 @@
      }
     ]
    },
+   "/apis/subresources.kubevirt.io/v1/namespaces/{namespace:[a-z0-9][a-z0-9\\-]*}/virtualmachineinstances/{name:[a-z0-9][a-z0-9\\-]*}/ssh/{interface:[a-z0-9][a-z0-9\\-]*}/{port:[0-9]+}": {
+    "get": {
+     "description": "Open a websocket connection to connect to SSH on the specified VirtualMachineInstance network interface and port.",
+     "operationId": "v1SSH/withInterfaceName",
+     "responses": {
+      "401": {
+       "description": "Unauthorized"
+      }
+     }
+    },
+    "parameters": [
+     {
+      "uniqueItems": true,
+      "type": "string",
+      "description": "The network interface on the VirtualMachineInstance the SSH connection should be forwarded to.",
+      "name": "interface",
+      "in": "path",
+      "required": true
+     },
+     {
+      "uniqueItems": true,
+      "type": "string",
+      "description": "Name of the resource",
+      "name": "name",
+      "in": "path",
+      "required": true
+     },
+     {
+      "uniqueItems": true,
+      "type": "string",
+      "description": "Object name and auth scope, such as for teams and projects",
+      "name": "namespace",
+      "in": "path",
+      "required": true
+     },
+     {
+      "uniqueItems": true,
+      "type": "string",
+      "description": "The SSH port on the VirtualMachineInstance.",
+      "name": "port",
```

Comment:
I think that this should be a parameter.

Reply:
I agree, but it's not possible (sadly).
Kubernetes drops all query params on websocket proxying (https://github.com/kubernetes/kubernetes/pull/90721) right now.

Reply:
I may misunderstand the issue but it looks like this is only an issue if you try to  open a websocket connection through `kubectl proxy`?

Reply:
I think it shows in proxy, but I had it as a param before and it got lost.
It seems like a general thing when proxying traffic through the apiserver.